### DISCUSSION TITLE
I'm non-technical, looking to build a directory website (kinda similar to 1800d2c.com).

### DISCUSSION THREAD

**Discussion by johnibasco** [webstudio-is/webstudio-community#159] [Q&A] [answered]
I'm building a specialty directory, similar to 1800d2c.com.
It's a directory consolidating every software, hardware, and service in the B2B cannabis space. (though I'm focusing on software as a start).

It will have:
3 segments (software, hardware, and services).
14 main categories (verticals within my industry)
35+ software sub-categories

It will have Free and Paid profiles as well for the vendors.
Also, a Filtering dock to help users navigate through categories (I'll post a screenshot).
![filtering dock](https://github.com/webstudio-is/webstudio-community/assets/164358063/967a48a5-600f-4c85-a0d5-a3c6fc777042)

Should I use Webstudio since I'll need a headless CMS for the listings? There's not a lot of resources to know about Webstudio for Directories, nor I have seen anyone build one. I would be highly grateful for some advice.

My current main option is Webflow + native CMS + Airtable (easy to handle for me, as a nocoder)
What should I go for?

**Comment by kof** [comment, marked as answer]
we have tutorials on that https://www.youtube.com/@getwebstudio

**Reply by johnibasco** [reply]
Awesome. Thanks!
